src/pages/my-events/[id].tsx

```diff
@@ -0,0 +1,94 @@
+// Page for repairers to view the repair requests associated with
+// a given event. This page is accessed via the my-events page.
+import { Fragment } from "react";
+import Image from "next/image";
+import { useRouter } from "next/router";
+import { Tab } from "@headlessui/react";
+
+import RequestView from "@/components/EventBox/Request-view";
+import LoadingSpinner from "@/components/UI/loading-spinner";
+import { useEvent } from "@/hooks/events";
+import { useRepairRequests } from "@/hooks/events";
+
+const Home = () => {
+  const {
+    query: { id: eventId }
+  } = useRouter();
+
+  const { isLoading, data: repairRequests } = useRepairRequests(
+    eventId as string
+  );
+  const { data: event } = useEvent(eventId as string);
+
+  return (
+    <div>
+      {/* HEADER BAR*/}
+      <div className="relative z-10 mt-2 flex w-full justify-center">
+        <Image
+          src="/images/repair_lab_logo.png"
+          alt="Repair Labs Logo"
+          width={90}
+          height={90}
+        />
+      </div>
+
+      <h1 className="relative z-10 mt-2 text-xl flex w-full justify-center">
+        {event ? event.name : <LoadingSpinner />}
+      </h1>
+      <hr className="mx-10" />
+
+      <div className="mt-4 relative flex-row items-center justify-center text-lg mx-10 bg-slate-200 rounded-lg">
+        {event ? (
+          <Tab.Group>
+            <Tab.List className="flex justify-center rounded-t-lg bg-slate-300 rounded-lg">
```

Comment:
color might be a bit too light use something a bit darker

Reply:
Currently the colour here is consistent with our other page.
We anticipate this page to undergo changes when issue 118 is resolved, and so likely this page is mostly for testing currently. We can remove it entirely if desired, we just pushed it as we spent some time on it before realising it was probably not part of our issue, and we needed a page for testing.